File: Assignment 7/Problem1.java
//Write a program which accept number from user and return the count of even digits.

/*
  Input : 2395
  Output : 1
  
  Input : 1018
  Output : 2
*/

import java.util.*;
public class Problem1
{
    public static void main(String [] arg)
    {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter Number : ");
        int iNo = sc.nextInt();
        int count = 0;

        while(iNo != 0)
        {
            int rem = iNo % 10;
            if(iNo%2 == 0)
            {
                count++;
            }
            iNo /= 10;
        }
        System.out.println(count);
    }
}

File: Assignment 3/problem1.c
// Write a program which accept one number from user and print that number of even numbers on screen.

// Input : 7
// Output : 2 4 6 8 10 12 14

#include <stdio.h>
void printEven(int iNo)
{
    if (iNo <= 0)
    {
        return;
    }
    for (int iCnt = 1; iCnt <= iNo; iCnt++)
    {
        printf("%d ", 2 * iCnt);
    }
}
int main()
{
    int iValue = 0;
    printf("Enter number: ");
    scanf("%d", &iValue);

    printEven(iValue);

    return 0;
}